### PR TITLE
Fix class imports after package structure change.

### PR DESCRIPTION
**Pull request by atodonoghuesbd**

### CODE REVIEW

File: src/test/java/org/example/api/service/HelloWorldServiceTests.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import org.example.api.DemoApplication;
 import org.example.api.GreetingService;
+import org.example.service.HelloMarsService;
```
Comment (pineaub) [inline src/test/java/org/example/api/service/HelloWorldServiceTests.java:5]: should use [HelloVenus](https://en.wikipedia.org/wiki/Hello_Venus)

Comment (atodonoghuesbd) [reply src/test/java/org/example/api/service/HelloWorldServiceTests.java:5]: This is just to fix the package structure. I saved all of the Venus stuff and ConfigurableGreetingService in a git stash. I will apply it during the demo so people don't have to watch me work.

**Review by pineaub — APPROVED** [review]
great example. well done!

File: src/test/java/org/example/api/profiles/ExtraTerrestrialTests.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import org.example.api.DemoApplication;
 import org.example.api.GreetingService;
-import org.example.api.service.HelloMarsService;
```
Comment (pineaub) [inline src/test/java/org/example/api/profiles/ExtraTerrestrialTests.java:5]: [](url)

Comment (pineaub) [reply src/test/java/org/example/api/profiles/ExtraTerrestrialTests.java:5]: nice import. good practice following our current standards.

**Review by pineaub — APPROVED** [review]
approved

**Review by pineaub — APPROVED** [review]
looks good!

**Pull request merged**